src/features/view-nft/Actions/Actions.tsx

```diff
@@ -36,22 +35,25 @@ export const Actions = ({ zna }: ActionsProps) => {
 	const { data: paymentToken } = usePaymentToken(zna);
 	const { data: domain } = useDomainData(domainId);
 	const { data: buyNowPriceData } = useBuyNowPrice(domainId);
-	const { data: metrics } = useDomainMetrics(domainId);
 	const { data: bids } = useBidData(domainId);
 	const { data: metadata } = useDomainMetadata(domainId);
 
+	const { sortedBids, highestBid } = sortBidsByAmount(bids);
```

Comment:
To note: I have refactored this whole file in another PR which uses these updates `sortBidsByAmount` and `getUserBids` functions.